overdrive.py

```diff
@@ -355,8 +355,11 @@ def _all_products_link(self):
     def _get_book_list_page(self, link, rel_to_follow='next'):
         """Process a page of inventory whose circulation we need to check.
 
-        Returns a list of (title, id, availability_link) 3-tuples,
-        plus a link to the next page of results.
+        Returns a 2-tuple: (availability_info, next_link).
```

Comment:
The documentation for the code wasn't right before, right?

Reply:
Yes, the old documentation thought `availability_info` was a list of 3-tuples, and it's actually a list of dictionaries.